.devcontainer/avatax/docker-compose.yml

```diff
@@ -6,14 +6,16 @@ services:
     build:
       context: ../..
       dockerfile: .devcontainer/avatax/Dockerfile
+    extra_hosts:
+      - "localhost:host-gateway"
     volumes:
       - "../..:/app"
       - "pnpm-store:/app/.pnpm-store"
 
   dynamodb:
     image: "amazon/dynamodb-local:latest"
     ports:
-      - 8000:8000
+      - "8000"
```

Comment:
If we change that I think we should also change scripts to setup dynamodb - currently they are requiring localhost:8000 - maybe we should globally change that to e.g 6667 or something similar?